doc/release-notes/release-notes-bucash1.2.0.0.md

```diff
@@ -0,0 +1,139 @@
+Release Notes for Bitcoin Unlimited Cash Edition 1.2.0.0
+=========================================================
+
+Bitcoin Unlimited Cash Edition version 1.2.0.0 is now available from:
+
+  <https://bitcoinunlimited.info/download>
+
+Please report bugs using the issue tracker at github:
+
+  <https://github.com/BitcoinUnlimited/BitcoinUnlimited/issues>
+
+This is a minor release version based of Bitcoin Unlimited compatible
+with the Bitcoin Cash specification you could find here:
+
+https://github.com/Bitcoin-UAHF/spec/blob/master/uahf-technical-spec.md
+
+
+Upgrading
+---------
+
+If you are running an older version, shut it down. Wait until it has completely
+shut down (which might take a few minutes for older versions), then run the
+installer (on Windows) or just copy over /Applications/Bitcoin-Qt (on Mac) or
+bitcoind/bitcoin-qt (on Linux).
+
+The first time you run this new version, your UTXO database will be converted
+to a new format. This step could take a variable amount of time that will depend
+on the performance of the hardware you are using.
+
+Downgrade
+---------
+
+In case you decide to downgrade from BUcash 1.2.0.0 to a previous version you
+will need to run the old release using `-reindex-chainstate` option so that the
+UTXO will be rebuild using the previous format.
+
```

Comment:
we can probably remove this though since you'd have to want to downgrade 2 revs which probably no-one would want to do.

Reply:
agreed